### DISCUSSION TITLE
我们这周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#419] [Announcements]
大家好，请注意，我们这周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）。

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Comment by ycjdixon** [comment]
新的测试版8.0.0依然抛弃WIN7系统，已经下载测试过了，安装后打不开。

**Reply by MaxMKMax** [reply]
@ycjdixon 嗯嗯. 我帮您反馈看看.

**Comment by Jumpsteelrail** [comment]
@Mike777Khoo 感谢！蓝灯恢复正常了

**Reply by MaxMKMax** [reply]
@Jumpsteelrail 非常开心听到您的反馈. 谢谢你们支持蓝等. 我们也希望你们用的欢心!

**Comment by wangmeng1991** [comment]
SOCKS5代理端口是多少 为什么这个版本看不到了 mac版

**Reply by MaxMKMax** [reply]
@wangmeng1991 
https://github.com/getlantern/lantern
您好! 您可以在这里看到Mac版本.

https://lantern.io/zh-Hans/faq
如果还遇到什么问题, 您可以参考上面的链接. 谢谢您.

**Comment by wangmeng1991** [comment]
还有 mac版app连接后能不能不要保留在dock栏 有点占地方

**Reply by MaxMKMax** [reply]
@wangmeng1991 您好，谢谢您的反馈

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by Lvfulongmy** [comment]
![Uploading image.png…]() 升级到桌面版之后 一直授权不成功 上面是错误日志 希望能有帮助

**Comment by loywong** [comment]
win10和macOS 11.7.8安装Preview版本，无法通过电子邮件授权设备，点击提交按钮后，邮箱无法收到邮件。

**Reply by MaxMKMax** [reply]
@loywong 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by loywong** [reply]
hit Send Report (点击发送报告)，提示错误：report_issue_error。在哪个目录下可以看到具体的log信息？

**Reply by loywong** [reply]
是否可以提供一个售后邮箱，可以帮我激活起其中一台设备，然后我通过此设备激活其他设备。

**Reply by MaxMKMax** [reply]
@loywong 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Comment by laplacemagic** [comment]
我安装新版本后闪退

**Comment by GithubLexi666** [comment]
用的Windows最新版，根本无法用邮箱登陆啊，之前的版本下载后浏览器根本打不开
![Uploading 微信图片_20240630020137.jpg…]()

**Comment by konaisland** [comment]
端口号，和全部代理都没有了啊，这两项很重要！

**Comment by jiang-31** [comment]
Hello, I want to use chatgpt, so I want to choose the node in the United States. How to choose?

**Reply by MaxMKMax** [reply]
@jiang-31 Hi, so far we have no option to choose the node in the US yet. It will be allocated to the best performance server. Sorry for the inconvenience caused. Thanks.

**Reply by jiang-31** [reply]
Hello, first of all, thank you very much for solving my payment problem. Unfortunately, I mainly need to use chatgpt, so I need to choose the VPN location, but now lantern does not have this function, so I have to trouble you to handle the refund for me.

**Reply by MaxMKMax** [reply]
@jiang-31 Hi Jiang, would you mind to try again the latest version again before we proceed to refund? Let's take a try on the latest version and give us one more chance to use our app. We truly appreciate and thankful if you choose not to refund. Thank you so much.

**Comment by Jameshening** [comment]
已经在手机端买了专业版，可以授权给桌面版吗？

**Reply by MaxMKMax** [reply]
https://lantern.io/zh-Hans/faq
@Jameshening 以上链接可以帮到您. 嗯嗯可以的.

**Comment by pppppppppppppjsjns** [comment]
可以看一下我的问题贴么？

**Reply by MaxMKMax** [reply]
@pppppppppppppjsjns 您好. 刚才在您的问题帖给您回复信息了.

**Comment by pppppppppppppjsjns** [comment]
发了图片给您了&nbsp; 能看的到么



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月8日(周四) 凌晨1:38
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 我们这周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本） (Discussion #419)




 
@pppppppppppppjsjns 您好. 刚才在您的问题帖给您回复信息了.

**Comment by ruochen800** [comment]


**Reply by MaxMKMax** [reply]
@ruochen800 您好. 谢谢您的反馈. 我会帮您反馈价格的事情, 看是哪里出错了还是第三方支付系统多收钱了. 您可以提供我支付单据吗? 我需要单据来帮您反馈给他们看. 谢谢!

关于授权的话基本上全部设备都可以的. 除了iOS系统. 以下链接有提供信息怎么授权. 希望可以帮到您.
https://lantern.io/zh-Hans/faq

**Reply by MaxMKMax** [reply]
@ruochen800 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@ruochen800 好的. 我帮您申请延长时间. 和技术团队反馈过了. 他们在修复当中. 跟汇率系统有关, 但我会帮您申请延长时间. 谢谢您支持蓝灯!非常荣幸有像您一样的用户能谅解我们.

**Reply by MaxMKMax** [reply]
@ruochen800 好的. 您可以删除了. 授权码我可以帮您和相关部门获得. 您等一下.

**Comment by MaxMKMax** [comment]
@ruochen800 
![image](https://github.com/user-attachments/assets/144ca8c0-82e4-471f-92b8-c097dfdbeb0d)

**Comment by whpublic2024** [comment]
交钱了不提供服务，你光发版本版本有啥用？

**Comment by ruochen800** [comment]


**Reply by MaxMKMax** [reply]
@ruochen800 

Hi Dear Ruochen, thanks for reaching us again. I had replied you on another post as well. We are happy to assist you on the matter or problem you facing.

For this case, you might need to have an authorization code for your device. Not sure if you can get the code? Please refer to this link for clearer information. If you are unable to get the code, feel free to reach me again and i will get the code from tech support to assist you on login on other devices.
https://lantern.io/zh-Hans/faq

(A) Sure i will help you to check which account for renewal. Will get back to you once the payment team is updating me.

(B) Regarding the overcharges issues was happened because of some technical issues from third parties exchange rate system. Regarding the additional charges, i would suggest we can add extra time for your pro account. I will get back to you after discussion with the team.

Thanks for being patiently wait us to solve the issues and appreciate you supporting Lantern. It's great to have you as our user.

Thank you so much and have a nice day!

**Reply by MaxMKMax** [reply]
@ruochen800 Hi, i think the team has added the extra time for extra charge of 15% fee for you. Did you see your account expired date being successfully extended?

**Comment by txwgoogol** [comment]
@Mike777Khoo 无法登录，不能使用了

**Comment by hy197** [comment]
如何续费
